datafusion/physical-plan/src/repartition/mod.rs

```diff
@@ -415,7 +415,7 @@ struct RepartitionMetrics {
     /// Time in nanos to execute child operator and fetch batches
     fetch_time: metrics::Time,
     /// Time in nanos to perform repartitioning
-    repartition_time: metrics::Time,
+    repart_time: metrics::Time,
```

Comment:
to me, the previous name is more readable. do we need this change?

Reply:
My rationale was so that the field name matched the display name of this metric (which is `repart_time`)

(a few [lines down ](https://github.com/apache/datafusion/pull/11478/files#diff-0886bee029ab49ae97b48450576c44717fd0300d8cff9edcde6308f1a40e0c3eR437) from here)

Another potential would be to rename the metric to `"repartition_time"` which would be a user facing change (which is probably fine). 

I think it would be nice to have them consistent. The exact value I have less of an opinion of

Reply:
Yeah, i agree that consistency is really desirable.  Therefore this PR is a good change (I will approve).
I would prefer that we change the metric name though. Readable naming is as valuable for code as it is for external metrics. I know it's a breaking change, but I don't know of  other way to improve the naming while avoiding the breakage.

Reply:
in https://github.com/apache/datafusion/pull/11703